spec/rubocop/cop/rspec/each_spec.rb

```diff
@@ -0,0 +1,80 @@
+RSpec.describe RuboCop::Cop::RSpec::Each do
+  subject(:cop) { described_class.new }
+
+  it 'flags `each` with an expectation' do
+    expect_violation(<<-RUBY)
+      it 'validates users' do
+        [user1, user2, user3].each { |user| expect(user).to be_valid }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer using the `all` matcher instead of iterating over an array.
+      end
+    RUBY
+  end
+
+  it 'flags `each` when expectation calls method with arguments' do
+    expect_violation(<<-RUBY)
+      it 'validates users' do
+        [user1, user2, user3].each { |user| expect(user).to be_an(User) }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer using the `all` matcher instead of iterating over an array.
+      end
+    RUBY
+  end
+
+  it 'ignores `each` with proc' do
+    expect_no_violations(<<-RUBY)
+      it 'validates users' do
+        [user1, user2, user3].each(&:to_s)
```

Comment:
Since this proc also doesn’t set an expectation, isn’t this case covered in the following spec (line 30)?